.changelog/19077.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+acl: Adds workload identity bind type and templated policy
```

Comment:
Nit: "Adds workload identity templated policy" sounds a bit better since templated policy is the bind type and not workload identity